parlai/mturk/tasks/acute_eval/example_script.py

```diff
@@ -44,9 +46,7 @@ def set_args():
     args['question'] = 'Who would you prefer to talk to for a long conversation?'
 
     args['num_conversations'] = int(
-        len(args['model_comparisons'])
-        * args['pairs_per_matchup']
-        / (args['task_description']['num_subtasks'] - 1)
+        args['pairs_per_matchup'] / max((args['subtasks_per_hit'] - 1), 1)
```

Comment:
much better

Reply:
i'm realizing now that this parameter, `pairs_per_matchup`, is a little ambiguous. Before, it indicated how many pairs _per model-matchup_ there were - that is, if there were 3 models (A,B,C) , we could have 100 pairs of comparisons for each combination (A,B), (B,C), (A, C), resulting in 300 pairs. 

Perhaps this argument should just be `num_matchup_pairs`.